homeassistant/components/tradfri/__init__.py

```diff
@@ -147,10 +154,20 @@ async def on_hass_stop(event):
             hass.config_entries.async_forward_entry_setup(entry, component)
         )
 
+    async def async_keep_alive(now):
+        try:
```

Comment:
To avoid the keep-alive after shutdown, add this check:

```suggestion
        if hass.is_stopping:
            return

        try:
```